davisinteractive/utils/scribbles.py

```diff
@@ -151,3 +151,39 @@ def is_empty(scribbles_data):
     scribbles = scribbles_data['scribbles']
     has_lines = [len(s) > 0 for s in scribbles]
     return not any(has_lines)
+
+
+def annotated_frames(scribbles_data):
+    """ Computes which frames have a scribble
+
+    # Arguments
+        scribbles_data (dict): Scribble in the default format
+    # Returns
+        list: Number of the frames that contain at least one scribble
+    """
+    frames_list = []
+    scribbles = scribbles_data['scribbles']
+    for ii, scribble_frame in enumerate(scribbles):
+        if scribble_frame:
+            frames_list.append(ii)
+    return frames_list
+
+
+def annotated_frames_object(scribbles_data, object_id):
+    """ Computes which frames have a scribble for a certain object
+
+    # Arguments
+        scribbles_data (dict): Scribble in the default format
+        object_id (int): Id of the object of interest
+    # Returns
+        dict: Number of the frames that contain at least one scribble
+    """
+    frames_list = []
+    scribbles = scribbles_data['scribbles']
+    for ii, scribble_frame in enumerate(scribbles):
+        if scribble_frame:
```

Comment:
This line can be omitted as it will be a list, and it is safe to make a for of an empty list (there will be no iteration).